notesapi/v1/search_indexes.py

```diff
@@ -12,6 +12,7 @@ class NoteIndex(indexes.SearchIndex, indexes.Indexable):
     created = indexes.DateTimeField(model_attr='created')
     updated = indexes.DateTimeField(model_attr='updated')
     tags = indexes.CharField(model_attr='tags')
+    data = indexes.CharField(use_template=True)
```

Comment:
My only thought is that I'd expect this to increase memory/disk usage, since we've not just added text and tags to the index but the composite text+tags field as well. Should we consider removing the individual text/tags fields from the index? @jimabramson